rex_xai/prediction.py

```diff
@@ -24,11 +25,11 @@ def __init__(
     def __repr__(self) -> str:
         if self.bounding_box is None:
             if self.is_passing():
-                return f"CLASS: {self.classification}, CONF: {self.confidence}"
+                return f"CLASS: {self.classification}, CONF: {self.confidence:.5f}"
```

Comment:
I think this is okay, since the test in `explanation_test.py` is using 6 decimal places plus/minus the tolerance value, and this is only showing 5. Maybe both should be reduced to an even lower precision - I'll probably revisit this once I've implemented additional tests with different parameters.